lib/Widgets/HeaderLabel.vala

```diff
@@ -8,32 +8,69 @@
  * HeaderLabel is a start-aligned {@link Gtk.Label} with the Granite H4 style class
  */
 public class Granite.HeaderLabel : Gtk.Widget {
+    /**
+     * The primary header label string
+     */
     public string label { get; construct set; }
 
+    /**
+     * Optional secondary label string displayed below the header
+     */
+    public string? secondary_text {
+        get; construct set; default = null;
+    }
+
     /**
      * Create a new HeaderLabel
      */
-    public HeaderLabel (string label) {
-        Object (label: label);
+    public HeaderLabel (string label, string? secondary_text = null) {
```

Comment:
that's an API break, a solution would be to have:
```vala
public HeaderLabel.with_secondary (string label, string secondary_text) {
```

Reply:
Ah yeah good point. In this case, another constructor probably isn't any better than just initializing with property:

```vala
new HeaderLabel.with_secondary (
    "A Header String",
    "A secondary description string"
);
```
vs
```vala
new HeaderLabel ("A Header String") {
    secondary_text = "A secondary description string"
};
```